sections/elixir.zsh

```diff
@@ -39,7 +39,7 @@ spaceship_elixir() {
 
   if [[ $elixir_version == "" ]]; then
     spaceship::exists elixir || return
-    elixir_version=$(elixir -v 2>/dev/null | grep "Elixir" --color=never | cut -d ' ' -f 2)
+    elixir_version=$(elixir -v 2>/dev/null | spaceship::grep "Elixir" | cut -d ' ' -f 2)
```

Comment:
Can't we just do this?

```zsh
grep "Elixir" --color=never 2>/dev/null
```

I am just not sure that making our own wrapper around grep is a good idea.

Reply:
We need to make it compatible with different `grep`s. Just handling error does not work.
Making our own wrapper for binaries is good for mocking command output too. It will facilitate testing.